Translation/Exporter/ExporterCollector.php

```diff
@@ -25,28 +33,23 @@ public function addExporter($id, ExporterInterface $exporter)
     }
 
     /**
-     * Returns an exporter that support the given format.
-     *
-     * @param string $format
-     * @return ExporterInterface
+     * @param $format
+     * @param $file
+     * @param $translations
+     * @return bool
+     * @throws \RuntimeException
      */
-    public function getByFormat($format)
+    public function export($format, $file, $translations)
```

Comment:
@94noni You never know. But if this should be merged to this version, there should not be something like that, you can create method export, that you should leave the old one (maybe with @deprecated).